### ISSUE TITLE
blog tutorial is not working after 2.7.4

### ISSUE THREAD

**Issue by hyoo** [cytoscape/cytoscape.js#1674]
I was following this tutorial and realized it works only on 2.7.3, where current stable is 2.7.13.

http://blog.js.cytoscape.org/2016/07/04/social-network/
http://blog.js.cytoscape.org/public/demos/twitter-graph/index.html

I figured that it is broken after 2.7.4 and seems like layout.run() is not working properly. (it started working after I change the layout part
```
from
options.layout.run();
to
cy.layout({name: 'concentric'}) 
```

I couldn't find any related discussion here, so I am asking here whether there was any change in the layout functions.

**Comment by hyoo** [comment]
I found a closed issue #1500, based on the discussion, here is a fix
```
-       var concentricLayout = cy.makeLayout({
+       var concentricLayout = {
                name: 'concentric',
                concentric: function(node){
                        return 10 - node.data('level');
@@ -40,7 +40,7 @@ document.addEventListener('DOMContentLoaded', function(){
                        return 1;
                },
                animate: false
-       });
+       };

...

-                       options.layout.run();
+                       cy.layout(options.layout);
```

**Comment by josephst** [comment]
Thanks for the heads up on this. My code there was definitely wrong; I've updated the code and tutorial to work properly and bumped the version of Cytoscape.js that's used in the example to 2.7.13. The blog should have the changes now. [Here's my changes](https://github.com/cytoscape/cytoscape.js-tutorials/commit/53172f901ee4c215a8735403f7d0d4478a832033) so you can easily see what's been updated.
